pkg/config/updater.go

```diff
@@ -24,6 +24,9 @@ func (u *ConfigUpdater) UpdateRemoteConfig(ctx context.Context, remote baseConfi
 	if err := u.UpdateDbConfig(ctx, remote.ProjectId, remote.Db); err != nil {
 		return err
 	}
+	if err := u.UpdateExperimentalConfig(ctx, remote.ProjectId, &remote.Experimental); err != nil {
```

Comment:
```suggestion
	if err := u.UpdateExperimentalConfig(ctx, remote.ProjectId, remote.Experimental); err != nil {
```

`experimental` block should always exist, so let's pass by value.